### PR TITLE
ART-6236: find-bugs:sweep: Add --cve-only option

### PR DESCRIPTION
**Pull request by vfreex**
This feature is requested by OpenShift QE to find CVE trackers against a release branch.

### CODE REVIEW

**Comment by rioliu-rh** [comment]
@joepvd @thiagoalessio could you help to review this PR, thanks

File: elliottlib/cli/find_bugs_sweep_cli.py [hunk 1]
```diff
@@ -77,14 +79,17 @@ def __init__(self):
                    '"extras" instead of the default "image", bugs filtered into "none" are not attached at all.')
 @click.option('--brew-event', type=click.INT, required=False,
               help='Only in sweep mode: SWEEP bugs that have changed to the desired status before the Brew event')
+@click.option("--cve-only",
+              is_flag=True,
```
Comment (Ximinhan) [inline elliottlib/cli/find_bugs_sweep_cli.py:83]: should add `required=False`

File: elliottlib/cli/find_bugs_sweep_cli.py [hunk 2]
```diff
@@ -197,7 +202,8 @@ async def get_bugs_sweep(runtime: Runtime, find_bugs_obj, brew_event, bug_tracke
 
 
 async def find_and_attach_bugs(runtime: Runtime, advisory_id, default_advisory_type, major_version,
-                               find_bugs_obj, output, brew_event, noop, count_advisory_attach_flags, bug_tracker):
+                               find_bugs_obj, output, brew_event, noop, count_advisory_attach_flags, bug_tracker,
+                               cve_only: bool):
```
Comment (Ximinhan) [inline elliottlib/cli/find_bugs_sweep_cli.py:206]: is the parameter `cve_only` used in function `find_and_attach_bugs`?

Comment (vfreex) [reply elliottlib/cli/find_bugs_sweep_cli.py:206]: No, it is a leftover that has been moved to the constructor of `FindBugsSweep`.
Will remove it in the next push.

**Pull request merged**